src/python/pants/backend/jvm/tasks/jar_publish.py

```diff
@@ -165,7 +166,7 @@ def write(self, target, path):
 
     target_jar = self._internaldep(self._as_versioned_jar(target), target)
     if target_jar:
-      target_jar = target_jar.extend(dependencies=dependencies.values())
+      target_jar = target_jar.extend(dependencies=list(dependencies.values()))
```

Comment:
Not sure if the call to `list()` is necessary. I think this is an override of `extend()`? Usually it doesn't allow for keyword args.